.github/workflows/docker-hub-push.yml

```diff
@@ -0,0 +1,108 @@
+# focuses on pushing pycytominer docker images to docker hub
+name: docker-push
+
+env:
+  LANG: "en_US.utf-8"
+  LC_ALL: "en_US.utf-8"
+  PYTHONIOENCODING: "UTF-8"
+  TARGET_PYTHON_VERSION: "3.9"
+
+on:
+  schedule:
+    # runs every Wednesday at 7 AM UTC
+    - cron: "0 7 * * 3"
+  # trigger for pushes to main
+  push:
+    branches: [main]
+  # trigger for published releases (distinct from drafts)
+  release:
+    types:
+      - published
+
+jobs:
+  docker-push:
+    runs-on: ubuntu-latest
+    steps:
+      # checks out selected files for docker image build testing
+      - name: Checkout selected files
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            build/docker/Dockerfile
+          sparse-checkout-cone-mode: false
+      - name: Download sdist artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: sdist
```

Comment:
I don't think this will work as intended because I don't think there will be an artifact attached to this workflow run. This would be another argument in favor of housing this action within the overall ci workflow because then we could get the artifact from a rerun of `build``.

https://github.com/actions/download-artifact?tab=readme-ov-file#download-artifacts-from-other-workflow-runs-or-repositories

Reply:
Thanks for the feedback here, will plan on moving towards the "build-test-push" model you mention in other comments.